client/src/components/icons/MailIcon.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import SvgIcon, { SvgIconProps } from '@mui/material/SvgIcon';
+// TODO: fills...
+export default function MailIcon(props: SvgIconProps) {
+  return (
+    <SvgIcon {...props}>
+      <svg viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+        <g clipPath="url(#clip0_3398_1868)">
+          <path
+            d="M19.924 15.506L17.019 19.379C16.9544 19.4648 16.8721 19.5358 16.7777 19.5872C16.6834 19.6386 16.5791 19.6691 16.4719 19.6768C16.3648 19.6845 16.2572 19.6691 16.1565 19.6317C16.0557 19.5943 15.9642 19.5358 15.888 19.46L14.388 17.96M8.25 15.75H2.25C1.85218 15.75 1.47064 15.592 1.18934 15.3107C0.908035 15.0294 0.75 14.6478 0.75 14.25V2.25C0.75 1.85218 0.908035 1.47064 1.18934 1.18934C1.47064 0.908035 1.85218 0.75 2.25 0.75H20.25C20.6478 0.75 21.0294 0.908035 21.3107 1.18934C21.592 1.47064 21.75 1.85218 21.75 2.25V9M21.411 1.3L13.267 7.564C12.6887 8.00886 11.9796 8.25006 11.25 8.25006C10.5204 8.25006 9.81128 8.00886 9.233 7.564L1.089 1.3M11.25 17.25C11.25 18.8413 11.8821 20.3674 13.0074 21.4926C14.1326 22.6179 15.6587 23.25 17.25 23.25C18.8413 23.25 20.3674 22.6179 21.4926 21.4926C22.6179 20.3674 23.25 18.8413 23.25 17.25C23.25 15.6587 22.6179 14.1326 21.4926 13.0074C20.3674 11.8821 18.8413 11.25 17.25 11.25C15.6587 11.25 14.1326 11.8821 13.0074 13.0074C11.8821 14.1326 11.25 15.6587 11.25 17.25Z"
+            stroke="currentColor"
+            fill="none"
+            strokeOpacity="1"
+            strokeWidth="1.5"
+            strokeLinecap="round"
+            strokeLinejoin="round"
+          />
+        </g>
+        <defs>
+          <clipPath id="clip0_3398_1868">
```

Comment:
This is the clip path I was talking about. Maybe you should check if multiple MailIcons on the same page work with this? Otherwise I guess it should be removed, but not sure if that would break this icon... 🤔